cross-modal-search/requirements.txt

```diff
@@ -1,7 +1,7 @@
-jina[hub, http]==2.0.0rc5
+jina[hub, http]==2.0.5
```

Comment:
```suggestion
jina[hub, http]~=2.0.0
```

Reply:
Addressed, please see [e11b980](https://github.com/jina-ai/examples/pull/672/commits/e11b980488645a32eac1fbe3a7c9ca27def8d850) for the change. Thank you.